examples/tutorials/nb_python/ECCV_2020_Interactivity.py

```diff
@@ -121,44 +122,37 @@ def make_cfg(settings):
             "resolution": [settings["height"], settings["width"]],
             "position": [0.0, settings["sensor_height"], 0.0],
             "orientation": [settings["sensor_pitch"], 0.0, 0.0],
+            "sensor_subtype": habitat_sim.SensorSubType.PINHOLE,
         },
         "depth_sensor_1st_person": {
             "sensor_type": habitat_sim.SensorType.DEPTH,
             "resolution": [settings["height"], settings["width"]],
             "position": [0.0, settings["sensor_height"], 0.0],
             "orientation": [settings["sensor_pitch"], 0.0, 0.0],
+            "sensor_subtype": habitat_sim.SensorSubType.PINHOLE,
         },
         "semantic_sensor_1st_person": {
             "sensor_type": habitat_sim.SensorType.SEMANTIC,
             "resolution": [settings["height"], settings["width"]],
             "position": [0.0, settings["sensor_height"], 0.0],
             "orientation": [settings["sensor_pitch"], 0.0, 0.0],
+            "sensor_subtype": habitat_sim.SensorSubType.PINHOLE,
         },
         # configure the 3rd person cam specifically:
         "color_sensor_3rd_person": {
             "sensor_type": habitat_sim.SensorType.COLOR,
             "resolution": [settings["height"], settings["width"]],
             "position": [0.0, settings["sensor_height"] + 0.2, 0.2],
             "orientation": np.array([-math.pi / 4, 0, 0]),
+            "sensor_subtype": habitat_sim.SensorSubType.PINHOLE,
         },
     }
 
-    sensor_specs = []
-    for sensor_uuid, sensor_params in sensors.items():
-        if settings[sensor_uuid]:
-            sensor_spec = habitat_sim.SensorSpec()
-            sensor_spec.uuid = sensor_uuid
-            sensor_spec.sensor_type = sensor_params["sensor_type"]
-            sensor_spec.resolution = sensor_params["resolution"]
-            sensor_spec.position = sensor_params["position"]
-            sensor_spec.orientation = sensor_params["orientation"]
-
-            sensor_specs.append(sensor_spec)
-
     # Here you can specify the amount of displacement in a forward action and the turn angle
     agent_cfg = habitat_sim.agent.AgentConfiguration()
-    agent_cfg.sensor_specifications = sensor_specs
-
+    agent_cfg.sensor_specifications = sut.make_sensor_specs_from_settings(
+        sensors, settings
+    )
```

Comment:
Example of multiple SensorSpec initialization using a dictionary. sut.make_sensor_specs(sensors) and sut.make_sensor_specs_from_settings(sensors, settings) creates the SensorSpecs under the hood

"sensor_subtype": habitat_sim.SensorSubType.PINHOLE is added as we need to make sure that the SensorSpec initialized is a CameraSensor